databricks/koalas/config.py

```diff
@@ -170,6 +170,16 @@ def validate(self, v: Any) -> None:
             lambda v: v in ('sequence', 'distributed', 'distributed-sequence'),
             "Index type should be one of 'sequence', 'distributed', 'distributed-sequence'.")),
 
+    Option(
+        key='compute.ordered_head',
+        doc=(
+            "'compute.ordered_head' sets whether or not to operate head with natural ordering. "
+            "Koalas doesn't guarantee the row ordering so `head` could return some rows from "
```

Comment:
No big deal but I wouldn't use abbreviation (doesn't -> does not) in the documentation.